### DISCUSSION TITLE
<x-app test='test'> component does not pass values properly

### DISCUSSION THREAD

**Discussion by groovey** [laravel/framework#34456] [General]
- Laravel Version: 8.0
- PHP Version: 7.3
- Database Driver & Version:

### Description:
Components does not properly pass parameters when using `<x-app test='test'>`

When trying to test out `<x-sample test='test'>` this works.

### Steps To Reproduce:

// not working when calling  `<x-app test='test'>`
./components/app.blade.php
@props(['test' => null])
{{ dump($test) }}

// working when calling `<x-sample test='test>`
@props(['test' => null])
{{ dump($test) }}

The only difference is <x-app> naming.

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. 

You can post support questions on our discussions tab instead so I'm moving it over there. I won't be able to help you out personally but others who are watching this repository can help you out.

Alternatively, you can post this question on one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)

Thanks.
